AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/SQLStatement.swift

```diff
@@ -22,3 +22,11 @@ extension SQLStatement {
         return []
     }
 }
+
+extension SQLStatement {
+
+    // `modelType` is deprecated, instead modelSchema should be used.
+    var modelType: Model.Type {
+        fatalError("Use of modelType is deprecated in SQLStatement")
```

Comment:
Let's add a bit more context here, along the lines of "`DataStoreStatement.modelType` is deprecated. `SQLStatement` is an internal type and there should be no references to `modelType`. If you encounter this error, please open a GitHub issue <link>."